### ISSUE TITLE
Release pre-trained VAE model and MAD-AD artifacts on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [farzad-bz/MAD-AD#1]
Hi @farzad-bz 🤗

I'm Niels from the Hugging Face open-source team. I came across your interesting work on Arxiv, "MAD-AD: Masked Diffusion Autoencoder for Unsupervised Brain Anomaly Detection," and noticed the code is available on GitHub.  We at Hugging Face are working to make research artifacts more discoverable.

Your README mentions that a pre-trained VAE model will be available soon.  Once this is released, would you consider hosting it, along with the trained MAD-AD model checkpoint, on the Hugging Face Hub? This would greatly improve their discoverability and allow researchers to easily use your method.

We can help you add metadata tags to your model card to improve discoverability and link them to your paper page on hf.co/papers.  For PyTorch models, the `PyTorchModelHubMixin` can simplify the uploading process ([https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin)).  It adds `from_pretrained` and `push_to_hub` methods for easy uploading and downloading. Otherwise, you can use `hf_hub_download`. You can find a general guide on model uploading here: [https://huggingface.co/docs/hub/models-uploading](https://huggingface.co/docs/hub/models-uploading).

If you're also open to it, sharing the preprocessed dataset (or a sample) used for your experiments on the Hub would be a valuable addition. This enables straightforward reproduction of results using the `datasets` library ([https://huggingface.co/docs/datasets/loading](https://huggingface.co/docs/datasets/loading)).

Let me know if you're interested or have any questions.

Kind regards,

Niels
ML Engineer @ Hugging Face 🤗
